consumers/hidden-test-consumer/src/hidden_test_consumer/hidden_test_process_pb2_grpc.py

```
@@ -0,0 +1,97 @@
+
+        Args:
+            channel: A grpc.Channel.
+        """
+        self.streamHiddenTestProcess = channel.stream_unary(
+                '/hidden_test_process.HiddenTestProcess/streamHiddenTestProcess',
+                request_serializer=hidden__test__process__pb2.ProcessRequest.SerializeToString,
+                response_deserializer=hidden__test__process__pb2.ProcessResponse.FromString,
+                _registered_method=True)
+
+
+class HiddenTestProcessServicer(object):
```

Ruff (0.8.2)

Class `HiddenTestProcessServicer` inherits from `object`

Remove `object` inheritance

(UP004)